zerver/tests/test_messages.py

```diff
@@ -2764,6 +2765,70 @@ def do_edit_message_assert_error(id_, unique_str, error):
         self.login(self.example_email("cordelia"))
         do_edit_message_assert_success(id_, 'D')
 
+    @mock.patch("zerver.lib.actions.send_event", return_value=None)
+    def test_edit_topic_public_history_stream(self, mock_send_event: Any) -> None:
```

Comment:
Using `` Any `` here is unnecessary, you can annotate `` mock_send_event `` with its actual type - `` mock.MagicMock ``